Sming/Arch/Esp8266/standard.hw

```diff
@@ -13,27 +13,27 @@
 	},
 	"partitions": {
 		"phy_init": {
-			"address": "0x003000",
+			"address": "self.device.size - 0x4000",
 			"size": "4K",
 			"type": "data",
 			"subtype": "phy",
 			"filename": "$(FLASH_INIT_DATA)"
 		},
 		"sys_param": {
-			"address": "0x004000",
+			"address": "self.device.size - 0x3000",
 			"size": "12K",
 			"type": "data",
 			"subtype": "sysparam"
 		},
 		"rf_cal": {
-			"address": "0x007000",
+			"address": "self.device.size - 0x5000",
 			"size": "4K",
 			"type": "data",
 			"subtype": "rfcal"
 		},
 		"rom0": {
-			"address": "0x008000",
-			"size": "992K",
+			"address": "0x002000",
+			"size": "0x0F8000",
```

Comment:
@mikee47 Thanks a lot for the changes. Would it be possible to change the value here back to a human readable size in K? At least for me it is easier to see the max size without going to another tool to calculate the size in K.